runtime/doc/vim_diff.txt

```diff
@@ -87,6 +87,8 @@ of these in your config by simply removing the mapping, e.g. ":unmap Y".
 	nnoremap <C-L> <Cmd>nohlsearch<Bar>diffupdate<Bar>normal! <C-L><CR>
 	inoremap <C-U> <C-G>u<C-U>
 	inoremap <C-W> <C-G>u<C-W>
+	xnoremap * y/\V<C-R>"<CR>
+	xnoremap # y?\V<C-R>"<CR>
```

Comment:
Not a blocker, but: this mapping is fragile. It will fail on multiline selections (which I use quite often), selections containing `/`, etc.... 

That's why I suggested shipping a robust function or some other mechanism that prepares the search input to be truly "literal": https://github.com/neovim/neovim/issues/15471#issuecomment-905110271

So many users will need to replace this "default mapping" with something that works better.